pulp_rpm/tests/unit/test_modulemd.py

```diff
@@ -90,8 +89,7 @@ def test_parse_modular_preserves_literal_unquoted_values_3285(tmp_path):
     with open(file_name, "w") as file:
         file.write(sample_file_data)
 
-    mock_obj = SimpleNamespace(path=str(file_name))
-    all, defaults, obsoletes = parse_modular(mock_obj)
+    all, defaults, obsoletes = parse_modular(str(file_name))
```

Comment:
You should be able to get rid of the `str()` around `file_name`, I'm not sure why that was here?

Reply:
The `str` is there because I was using a `Path` object before. Will remove it